packages/opentelemetry/src/propagator.ts

```diff
@@ -196,17 +197,13 @@ export function makeTraceState({
   parentSpanId?: string;
   dsc?: Partial<DynamicSamplingContext>;
   sampled?: boolean;
-}): TraceState | undefined {
-  if (!parentSpanId && !dsc && sampled !== false) {
-    return undefined;
-  }
-
+}): TraceState {
   // We store the DSC as OTEL trace state on the span context
   const dscString = dsc ? dynamicSamplingContextToSentryBaggageHeader(dsc) : undefined;
 
-  const traceStateBase = parentSpanId
-    ? new TraceState().set(SENTRY_TRACE_STATE_PARENT_SPAN_ID, parentSpanId)
-    : new TraceState();
+  // We _always_ set the parent span ID, because we can infer from the existence of this if we should use this
```

Comment:
m: This comment is a bit confusing to read because of all of `this`. Can we replace `this` with actual objects? 😄

Reply:
I updated the comment!